common/lib/xmodule/xmodule/modulestore/tests/django_utils.py

```diff
@@ -198,6 +198,49 @@ def xml_store_config(data_dir, source_dirs=None):
 )
 
 
+class ModuleStoreClassCleanupTestCase(TestCase):
+    """
+    Sometimes you want to create a course and know it will stick around between
+    setUp()s.
+
+    Note that if you create any Django ORM data in your setUpClass() methods,
+    you must manually clean it up in your tearDownClass(). Django's built-in
+    TestCase class (which this class inherits from) will cleanup the database
+    between tests on databases that support transactions, but it cannot clean
+    up models created in setUpClass(). In Django 1.8, we will be able to use
+    setUpTestData() for this.
```

Comment:
Yeah. Funny note though -- once the shared modulestore init is done, the DjangoORM resetting becomes the tall straw in a number of the test cases. I briefly tried to emulate some of Django 1.8's `setUpTestData` functionality, but the code uses transaction methods that changed from 1.4, and I figured we'd just wait for the upgrade. But it's exciting as the Next Thing to put on the chopping block.